angulardart.dev -> angulardart.xyz

kwalrath opened the pull request:
Fixes #3378 

The community has started updating the AngularDart docs, as we were hoping they would (see https://angulardart.xyz and #3378 for details). How about we point to the community site?

Staged:
https://kw-dartlang-3.web.app/web

Btw, a lot of pages still refer to AngularDart. The following ones use the `{{site.angulardart}}` macro (and thus would now point to angulardart.xyz):

/web (as listed above)
/tools/dartdevc
/tools/webdev
/tools/pub/obsolete
/tutorials/web/low-level-html
/tutorials/web/low-level-html/connect-dart-html
/web/dart-2
/web/libraries
/faq
/guides/json
/guides/language/effective-dart/design
/guides/testing

timsneath commented:
This is exciting to see! 

I think we should probably scrub through those references and remove those that are not core to Dart itself. For example, https://dart.dev/tools/webdev#test shouldn't point to .xyz. 

In general, the philosophy I'd have is that this package should have the same level of prominence on dart.dev as any other top-class community package (but not _more_ -- it shouldn't be confusing to people as to whether this is part of the Google-supported offering or not). 

That https://dart.dev/web page is particularly in need of a scrub, since it looks like Google guidance, and that guidance seems pretty dated in the light of https://flutter.dev/web.

kwalrath commented:
OK, I've updated the files. ptal @timsneath.

The following pages no longer use the site.angulardart macro:
/web
/tools/dartdevc
/tools/webdev
/tools/pub/obsolete
/tutorials/web/low-level-html
/tutorials/web/low-level-html/connect-dart-html
/faq

The following pages _do_ still use the macro, but it seems justifiable:
/web/dart-2
/web/libraries
/guides/json
/guides/language/effective-dart/design
/guides/testing

The following additional pages have the string Angular visible, in a valid historical context:
/guides/whats-new
/resources/books

I've updated the staging server:
https://kw-dartlang-3.web.app/guides/whats-new

timsneath commented:
Looks great! Nice job!

GZGavinZhao commented:
@kwalrath Thank you for your effort! This looks really cool, but I hope that we can still use the macro in the page `/web`, perhaps with something similar to the first change that mentions AngularDart and stating it's not an official Google product. This will be more useful and convenient to users who are looking for a Dart web framework other than Flutter.

timsneath commented:
My hope is that over time we can get it to something akin to "Flutter Favorite" status -- that is, as the community version matures, it becomes obviously durable and highly recommended just like packages such as hive or redux. And yeah, the goal isn't to bar it from being mentioned, just to make sure that we don't confuse folk into thinking it has the same level of official support as Dart itself.
